### PR TITLE
Bump DJL version to 0.27.0

### PR DESCRIPTION
**Pull request by siddvenk**
bump to 0.27.0 and update README with latest release

### CODE REVIEW

**Review by sindhuvahinis — APPROVED** [review]

**Comment by siddvenk** [comment]
we need to release patch 0.26.1 to fix a bug

**Comment by siddvenk** [comment]
a fix was raised in djl-serving to address the issue - no 0.26.1 patch needed here

File: README.md [hunk 1]
```diff
@@ -110,7 +111,7 @@ The following pseudocode demonstrates running training:
 * [0.2.1](https://github.com/deepjavalibrary/djl/releases/tag/v0.2.1) ([Code](https://github.com/deepjavalibrary/djl/tree/v0.2.1))
 * [0.2.0 Initial release](https://github.com/deepjavalibrary/djl/releases/tag/v0.2.0) ([Code](https://github.com/deepjavalibrary/djl/tree/v0.2.0))
 
-The release of DJL 0.26.0 is planned for January 2024.
+The release of DJL 0.26.0 is planned for February/March 2024.
```
Comment (frankfliu) [inline README.md:114]:
```suggestion
The release of DJL 0.27.0 is planned for March 2024.
```

Comment (siddvenk) [reply README.md]: good catch

**Pull request merged**